src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -123,13 +127,16 @@ private void buildCopyFromModal(List<CourseAttributes> courses, String courseIdF
                                                                              instructors, newFeedbackSession));
     }
 
-    private void buildFsList(String courseIdForNewSession, List<FeedbackSessionAttributes> existingFeedbackSessions,
-                             Map<String, InstructorAttributes> instructors, String feedbackSessionNameForSessionList) {
+    private void buildFsList(String courseIdForHighlight, List<FeedbackSessionAttributes> existingFeedbackSessions,
+                             Map<String, InstructorAttributes> instructors, String feedbackSessionNameToHighlight) {
```

Comment:
Should these variables be standardised with `ToHighlight` or `ForHighlight`?
Or is there an actual distinction between the two?